sycl/cmake/modules/UnifiedRuntimeTag.cmake

```diff
@@ -1,7 +1,7 @@
-# commit a9c7aef861988d4b8421f3e760e63f4e3da9398d (HEAD, origin/main, origin/HEAD)
-# Merge: dbd168c 3142142
-# Author: aarongreig <aaron.greig@codeplay.com>
-# Date:   Mon Oct 28 15:56:46 2024 +0000
-#    Merge pull request #2152 from Bensuo/fabio/binary_update_fix
-#    Fix command-buffer binary update implementation
-set(UNIFIED_RUNTIME_TAG a9c7aef861988d4b8421f3e760e63f4e3da9398d)
+# commit e3d127d69061c942b37c8e18b876281f5a9b3857
```

Comment:
This is no longer required. UR has been already updated.

Reply:
OK thanks, I've merged sycl branch into this.